demisto_sdk/commands/common/hook_validations/integration.py

```diff
@@ -1720,6 +1720,7 @@ def is_valid_py_file_names(self):
             "CommonServerPython.py",
             "CommonServerUserPython.py",
             ".vulture_whitelist.py",
+            "MicrosoftApiModule.py",
```

Comment:
what happens when a dev changes the actual `MicrosoftApiModule.py` file? is it not validated?

Reply:
I don't think it should affect anything.
This is an integration validation, the actual MSAPIModule is a script.

Reply:
cool, nice catch, please add this comment to the list